platform/ios/PolyPodApp/PolyPodUITests/XCUIApplicationExtensions.swift

```diff
@@ -30,6 +30,11 @@ extension XCUIApplication {
                 "\(showDeveloperFeatures)"
             ]
         }
+
+        // This makes sure an already running app is first made inactive
+        XCUIDevice.shared.press(.home)
+        sleep(1)
```

Comment:
Best workaround I could (so far) come up with for preferences not being saved in the tests, because `didEnterBackgroundNotification` doesn't seem to arrive. I tried `willTerminateNotification` and `willResignActiveNotification`, but that doesn't seem to work for saving preferences either. The breakpoints I set are quickly activated, but I can't actually debug it.